pt-BR/getting_started.md

```diff
@@ -240,587 +238,516 @@ navegue para <http://localhost:3000>.  Você deve ver a página padrão com info
-----------------------
-
-Agora que você já viu como criar um *controller*, uma *action*, e uma *view*, vamos criar algo um pouco mais relevante.
+### Configuração da Página Inicial da Aplicação
 
-Na aplicação do Blog você irá criar um novo *resource*. *Resource* é um termo utilizado para uma coleção de objetos similares, como artigos, pessoas ou animais.
-Você pode criar, visualizar, editar e deletar dados de um *resource* e essas ações são definidas como operações *CRUD*.
+No momento, <http://localhost:3000> ainda exibe "Yay! Você está no Rails!".
+Vamos mostrar nosso "Olá, Rails!" texto em <http://localhost:3000> também.
+Para fazer isso, vamos adicionar uma rota que mapeia o caminho raiz (*root path*) da nossa aplicação para o _controller_ e _action_ apropriados.
 
-O Rails te fornece um método `resources` que pode ser usado para declarar um recurso padrão *REST*. Você precisa adicionar o *article resource* no `config/routes.rb` e o arquivo ficará como a seguir:
+Vamos abrir o arquivo `config/routes.rb`, e adicionar o a rota `root` no começo do bloco `Rails.application.routes.draw`:
 
 ```ruby
 Rails.application.routes.draw do
-  get 'welcome/index'
-
-  resources :articles
+  root "articles#index"
 
-  root 'welcome#index'
+  get "/articles", to: "articles#index"
 end
 ```
 
-Se você executar `rails routes`, você verá que foram definidas rotas para todas as *actions* padrão *RESTful*.
-O significado do prefixo da coluna (e de outras colunas) será visto mais adiante, mas por enquanto, observe que o Rails entende `article` de forma singular e faz o uso significativo da distinção.
+Agora podemos ver a mensagem "Olá, Rails!", quando visitamos <http://localhost:3000> confirmando que a _route_ `root` também mapeia para a _action_ `index` de `ArticlesController`.
 
-```bash
-$ rails routes
-       Prefix Verb   URI Pattern                  Controller#Action
-welcome_index GET    /welcome/index(.:format)     welcome#index
-     articles GET    /articles(.:format)          articles#index
-              POST   /articles(.:format)          articles#create
-  new_article GET    /articles/new(.:format)      articles#new
- edit_article GET    /articles/:id/edit(.:format) articles#edit
-      article GET    /articles/:id(.:format)      articles#show
-              PATCH  /articles/:id(.:format)      articles#update
-              PUT    /articles/:id(.:format)      articles#update
-              DELETE /articles/:id(.:format)      articles#destroy
-         root GET    /                            welcome#index
-```
+TIP: Para mais informações sobre roteamento, consulte [Roteamento do Rails de Fora para Dentro](routing.html).
 
-Na próxima seção, você adicionará a funcionalidade para criar e visualizar novos artigos (articles) em sua aplicação. Este é o "C" e o "R" do *CRUD*: *create* (criação) e *read* (leitura). O formulário para fazer isso ficará assim:
 
-![The new article form](images/getting_started/new_article.png)
+MVC and You
```

Comment:
Daqui pra baixo mudou bastante né. Vamos ter que rever a tradução de novo. Acredito que seja melhor atualizar e, depois de mergeado, re-traduzir. Posso fazer isso quando for mergeado :smile:

Reply:
Exatamente, essa parte mudou quase tudo que não consegui fazer um DE-PARA da antiga pra nova, então teria que ser retraduzido 😢